operate/client/src/App/ProcessInstance/BottomPanel/VariablePanel/tests/placeholder.test.tsx

```diff
@@ -140,7 +139,9 @@ describe('VariablePanel', () => {
     mockFetchProcessDefinitionXml().withSuccess(
       mockProcessWithInputOutputMappingsXML,
     );
-    mockFetchProcessInstanceListeners().withSuccess(noListeners);
+    mockSearchJobs().withSuccess({items: [], page: {totalItems: 0}});
+    // FIXME: Can we optimize the rendering so jobs are only queries once for listeners?
```

Comment:
Generally we create an issue to not leave floating comments like this

Reply:
Oh I totally forget about that one. Let me make a small PR for it.